.github/workflows/lint.yml

```diff
@@ -77,4 +58,3 @@ jobs:
         env:
```

Comment:
We can simplify the jobs, as we had a special case if it was running in store and not in store,